### ISSUE TITLE
Report 1 Command failed: /usr/local/bin/swiftlint --quiet --reporter json --config /Users/fred/Dropbox/projects/Keychain/.swiftlint.ymlNo lintable files found at paths: ''

### ISSUE THREAD

**Issue by fappelman** [vknabel/vscode-swiftlint#6]
`Error: Command failed: /usr/local/bin/swiftlint --quiet --reporter json --config xxxxxx/Keychain/.swiftlint.yml
No lintable files found at paths: ''

	at ChildProcess.exithandler (child_process.js:303:12)
	at ChildProcess.emit (events.js:200:13)
	at maybeClose (internal/child_process.js:1021:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)`

Note that I changed the leading patch to `xxxxxxx` in this error report.

If I run the same command in the terminal I get this:

```bash
/usr/local/bin/swiftlint --quiet --reporter json --config xxxxxxx/Keychain/.swiftlint.yml
[

]
```

**Comment by vknabel** [comment]
This issue has been fixed in [1.0.4](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.0.4).
If your issue still persists, feel free to reopen this issue!

**Comment by fappelman** [comment]
Thanks.
